Formula/c/cmake.rb

```diff
@@ -8,11 +8,12 @@ class Cmake < Formula
   license "BSD-3-Clause"
   head "https://gitlab.kitware.com/cmake/cmake.git", branch: "master"
 
-  # The "latest" release on GitHub has been an unstable version before, so we
-  # check the Git tags instead.
```

Comment:
> What would be the benefit of doing that as opposed to tracking the GitHub latest?

@osalbahr this is the reason why the `GitHubLatest` strategy is not used. "Unstable" means beta, release candidates, etc.

> I don’t know if the website had 3.27.4 before or after the repo’s 3.27.4 tag was marked latest.

After, most likely. The website links to GitHub downloads.

Reply:
Oh I see now. Yeah I agree then that https://cmake.org/download is probably the most reliable way of knowing the latest stable version. I don’t know how to write the regex for that, though.

Reply:
Yes, I think announcing the version on the website is more deliberate, in addition to what `@ZhongRuoyu` said. Here's a suggestion:
```
livecheck do
  url "https://cmake.org/download/"
  regex(/href=.*?cmake[._-]v?(\d+(?:\.\d+)+)\.t/i)
end
```